app/models/mixins/relationship_mixin.rb

```diff
@@ -366,12 +454,20 @@ def subtree_rels(*args)
 
   # Returns a list of all records in the record's subtree
   def subtree(*args)
-    Relationship.resources(subtree_rels(*args)) # TODO: Prevent preload of self which is in the list
+    if @relationship_type == :genealogy
+      call_ancestry_method(:subtree, binding)
+    else
+      Relationship.resources(subtree_rels(*args)) # TODO: Prevent preload of self which is in the list
```

Comment:
wondering if this affects us.

I think this TODO was implemented. Wondering if we need this for ancestry